src/growth_region.h

```diff
@@ -90,7 +90,7 @@ class GrowthRegion : public cyclus::Region,
     "description is comprised of a function type and associated parameters. "\
     "\n\n"                                       \
     "  * Start times are inclusive. For a start time :math:`t_0`, the demand "\
-    "function is evaluated on :math:`[t_0, \infty)`."                     \
+    "function is evaluated on :math:`[t_0, infinity)`."                     \
```

Comment:
Interesting... I'm pretty sure the previous was chosen to enable LaTeX based rendering.... since this documentation is used in various places, that may be true in the minority of circumstances.  Interesting to think this through, though..